stable/pod-gpu-metrics-exporter/values.yaml

```diff
@@ -13,3 +13,37 @@ dcgmExporter:
   image:
     repository: "nvidia/dcgm-exporter"
     tag: "1.4.6"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+rbac:
+  create: true
+
+resources: {}
```

Comment:
I don't see this used anywhere. Please don't add to values if it's not being referred to anywhere

Reply:
from now `resources` block is present

Reply:
Great!